src/map/clif.cpp

```diff
@@ -6584,6 +6591,16 @@ void clif_status_change(struct block_list *bl, int type, int flag, t_tick tick,
 	clif_status_change_sub(bl, bl->id, type, flag, tick, val1, val2, val3, ((sd ? (pc_isinvisible(sd) ? SELF : AREA) : AREA_WOS)));
 }
 
+void clif_body_size(struct block_list *bl, int val1) {
+	map_session_data *sd = NULL;
+
+	nullpo_retv(bl);
+
+	sd = BL_CAST(BL_PC, bl);
+
+	clif_status_change_sub(bl, bl->id, 1421, 1, 9999, val1, 0, 0, ((sd ? (pc_isinvisible(sd) ? SELF : AREA) : AREA_WOS)));
```

Comment:
Add check to use right EFST
`clif_status_change_sub(bl, bl->id, (bl->type == BL_MOB) ? EFST_CHANGE_SIZE_MONSTER : EFST_CHANGE_SIZE, 1, 9999, val1, 0, 0, ((sd ? (pc_isinvisible(sd) ? SELF : AREA) : AREA_WOS)));`